src/oncall/ui/static/js/oncall.js

```diff
@@ -574,7 +635,11 @@ var oncall = {
             router.updatePageLinks();
           })
           .on('typeahead:selected', function(){
-            router.navigate('/team/' + $(this).val());
+            var ttPath = $input.parents(self.data.searchForm).find('.tt-cursor a').attr('href');
+            if (typeof ttPath == 'undefined'){
```

Comment:
Check with @shoxv, but I think you want `=== undefined`. I'm not sure what this event handler is doing, so I'll make @shoxv review it since he said he helped with it :P.